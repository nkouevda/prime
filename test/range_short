#!/usr/bin/env bash

# Nikita Kouevda
# 2014/12/19

bin/prime -rs 1000000
