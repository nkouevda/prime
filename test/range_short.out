78498 of 1000000 (7.849800%) in [0, 1000000)
